pom.xml

```diff
@@ -67,14 +69,31 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>analysis-core</artifactId>
-            <version>1.71</version>
-            <type>hpi</type>
+            <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
Didn't have time to check it, but thanks for confirming there was an issue in 1.73.

3.1 has been released about 45 minutes ago. Should be available through the update centre in the next hours.